SConstruct

```diff
@@ -79,6 +79,7 @@ def run_checks(env):
     if config.CheckHeader('stdatomic.h'):
         env.AppendUnique(CPPDEFINES=['HAVE_STDATOMIC=1'])
 
+    env.AppendUnique(CPPDEFINES=['CRT_PROC_RAW_STRUCT=1'])
```

Comment:
why do we have two versions?   It seems we only two version a couple of things but you've removed a fair amount of old code.